angr/analyses/decompiler/structured_codegen.py

```diff
@@ -177,21 +177,21 @@ def mapper(chunks, posmap, stmt_posmap, insmap):
                     elif isinstance(obj, CExpression) and hasattr(obj, 'tags') and \
                             obj.tags is not None and 'ins_addr' in obj.tags:
 
-                        # assure that variables are only placed once in the GUI at the defenition of each var
+                        # assure that variables are only placed once in the GUI at the defn of each var
```

Comment:
Let's spell out these words when writing comments. There is no need to abbreviate.

Reply:
kk